tests/pyccel/test_pyccel.py

```diff
@@ -235,36 +225,10 @@ def pyccel_test(test_file, dependencies = None, compile_with_pyccel = True,
     lang_out = get_lang_output(get_exe(test_file))
     compare_pyth_fort_output(pyth_out, lang_out, output_dtype)
 
-
-#==============================================================================
-# PYTEST MODULE SETUP AND TEARDOWN
-#==============================================================================
-def setup():
-    teardown()
-
-#------------------------------------------------------------------------------
-def teardown(path_dir = None):
-    if path_dir is None:
-        path_dir = os.path.dirname(os.path.realpath(__file__))
-
-    for root, _, files in os.walk(path_dir):
-        for name in files:
-            if name.startswith(".coverage"):
-                shutil.copyfile(os.path.join(root,name),os.path.join(os.getcwd(),name))
-
-    files = os.listdir(path_dir)
-    for f in files:
-        file_name = os.path.join(path_dir,f)
-        if f == "__pyccel__":
-            shutil.rmtree( file_name )
-        elif not os.path.isfile(file_name):
-            teardown(file_name)
-        elif not f.endswith(".py") and not f.endswith(".pyh") and not f.endswith(".pyccel"):
-            os.remove(file_name)
-
 #==============================================================================
 # UNIT TESTS
 #==============================================================================
+@pytest.mark.xdist_incompatible
 def test_relative_imports_in_project(language):
```

Comment:
Why is this test incompatible with `pytest-xdist`? It creates its own temporary directories, doesn't it?

Reply:
Sorry I saw the same file name but missed that `test_relative_imports_in_project` and `test_absolute_imports_in_project` are operating in different folders

Reply:
Fixed in 74065a32